golem/task/taskserver.py

```diff
@@ -388,6 +388,12 @@ def add_task_header(self, task_header: dt_tasks.TaskHeader) -> bool:
                 task_header.signature,
             )
             return False
+        if task_header.deadline < time.time():
+            logger.info(
+                "Task's deadline already in the past. task_header: %r",

```

Comment:
```suggestion
                "Task's deadline already in the past. task_header=%r",
```

https://github.com/golemfactory/golem-internal/wiki/Logging-guidelines

Reply:
okay, I'll try to remember that, although in this case, I'd say it's not worth a separate commit ...